RCAIDE/Library/Components/Energy/Lines/Jets.py

```diff
@@ -0,0 +1,48 @@
+# RCAIDE/Library/Components/Energy/Networks/Jets.py
```

Comment:
For whatever reason I can't make comments any where I want. In regards to  Components/Airfoil_Data, delete this folder from Components, this could get out of hand.